src/System.Device.Gpio/System.Device.Gpio.csproj

```diff
@@ -39,6 +39,10 @@
     </Reference>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' != 'netstandard2.0'">
```

Comment:
Shouldn't this condition be the other way around? Right now it is compiling the file on net5.0 but not on NS2.0

Reply:
Discussed offline, we will instead move this type to iot.device.bindings and the binding itself and try to make it internal.